src/DynamicData/List/Linq/AddKeyEnumerator.cs

```diff
@@ -55,14 +55,23 @@ public AddKeyEnumerator(IChangeSet<TObject> source, Func<TObject, TKey> keySelec
 
                 case ListChangeReason.Replace:
                     {
-                        // replace is a remove and add
+                        // replace is a remove and add, IFF the keys do not match
```

Comment:
`IF` not `IFF`?

Reply:
if, and only if, (https://en.wikipedia.org/wiki/If_and_only_if)

Reply:
Probably worth not using abbreviations in that case.